src/tooltip/test/tooltip.spec.js

```diff
@@ -67,55 +67,73 @@ describe('tooltip', function() {
       element: '<a data-animation="am-flip-x" bs-tooltip="tooltip">hover me</a>'
     },
     'options-placement-top': {
-      element: '<a data-placement="top" bs-tooltip="tooltip">hover me</a>'
+      element: '<a data-placement="top" bs-tooltip="tooltip" data-viewport="null">hover me</a>'
```

Comment:
@mgcre the bootstrap default is to have the body as the viewport.  If the viewport wasn't null for the standard placements and auto placements, things would shift causing the tests to have to account for it.  I thought it would be easier to test those placements with one less variable.